File: 1_iniciante/java/1008.java
/**
 * @source Beecrowd
 * @problem 1008 - Salary
 * @subject Sequential
 * 
 * @author Letícia Cruz
 * @date 2025-07-05
 */

import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        int numFuncionario = scanner.nextInt();
        int horasTrabalhadas = scanner.nextInt();
        double valorHora = scanner.nextDouble();

        double salario = horasTrabalhadas * valorHora;

        System.out.println("NUMBER = " + numFuncionario);
        System.out.printf("SALARY = U$ %.2f\n", salario);

        scanner.close();
    }
}

File: 1_iniciante/javascript/1008.js
/**
 * @source Beecrowd
 * @problem 1008 - Salary
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2020-10-01
 */

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var numero = parseInt(lines.shift());
var horas = parseInt(lines.shift());
var salariohoras = parseFloat(lines.shift());

var salariototal = horas * salariohoras;

console.log("NUMBER = " + numero);
console.log("SALARY = U$ " + salariototal.toFixed(2));
